recipes/expat/all/conanfile.py

```diff
@@ -40,10 +40,12 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        if not is_msvc(self):
-            del self.settings.compiler.libcxx
+        del self.settings.compiler.libcxx
```

Comment:
my bad, it must be protected with try/except for CONAN_V2 mode: https://docs.conan.io/en/1.51/migrating_to_2.0/recipes.html#settings